Client/Frontend/Strings.swift

```diff
@@ -16,6 +16,11 @@ private func applicationBundle() -> Bundle {
     return Bundle(url: applicationBundleURL) ?? bundle
 }
 
+extension Strings {
+    public static let OKString = NSLocalizedString("OK", comment: "OK button")
```

Comment:
Thank you!! I was looking for a generic version of this the other day. We should probably do a follow-up to clean up any dupes here.